src/hooks/screen/services/chatScreenHooks.js

```diff
@@ -1,9 +1,10 @@
 import {useQuery} from 'react-query';
+
 import ChatSchema from '../../../database/schema/ChatSchema';
 
 export function useGetAllMessage(payload, options) {
   return useQuery(
-    ['getAllMesage', payload],
+    [`getAllMesage-${payload?.selectedChannelId}`, payload],
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change made to the key in `useQuery` is a good one. It ensures that each channel has its own unique cache, which can help prevent data from different channels from getting mixed up. However, there's a typo in the string 'getAllMesage'. It should be 'getAllMessage'.
```diff
-    [`getAllMesage-${payload?.selectedChannelId}`, payload],
+    [`getAllMessage-${payload?.selectedChannelId}`, payload],
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->